pkgs/development/python-modules/dm-sonnet/default.nix

```diff
@@ -50,5 +50,11 @@ buildPythonPackage rec {
     homepage = "https://github.com/deepmind/sonnet";
     license = licenses.asl20;
     maintainers = with maintainers; [ onny ];
+    broken = true; # at 2022-09-23
+      #  > WARNING: Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by
+      #      'NewConnectionError('<pip._vendor.urllib3.connection.HTTPSConnection object at 0x7ffff5950cd0>:
+      #        Failed to establish a new connection: [Errno -3] Temporary failure in name resolution')': /simple/etils/
+      #  > ERROR: Could not find a version that satisfies the requirement etils[epath] (from versions: none)
```

Comment:
```suggestion
```

This boilds down to etils missing and the comment goes above the line it belongs to.

Reply:
Fixed.